client/www/scripts/modules/datasource/datasource.directives.js

```diff
@@ -26,7 +27,12 @@ Datasource.directive('slDatasourceEditorForm', [
         scope.$watch('datasource.connectionTestResponse', function(response) {
           React.renderComponent(DatasourceEditorView({scope:scope}), el[0]);
         }, true);
-
+        $timeout(function() {
+          window.setScrollView('[data-id="DatasourceEditorInstanceContainer"]');
```

Comment:
I don't see `window.setScrollView()` documented anywhere. Is this a native JS function?